<launch>
    <param name="robotless_debug" type="bool" value="true" />
    <node name="panda_gui" pkg="rqt_gui" type="rqt_gui" args="-s panda_gui" output="screen"/>
</launch>
